src/noisepy/seis/application_modules/esyn_utils.py

```diff
@@ -0,0 +1,122 @@
+import math
+from typing import Tuple
+
+import numpy as np
+import pyasdf
+
+### -----
+# These scripts are aim to perform the 2-D radiative transfer equation
+# for scalar waves (Shang and Gao 1988; Sato 1993),
+# assuming of isotropic scattering and source radiation in infinite medium
+# to calculate synthesized energy densities Esyn.
+### -----
+
+
+### -----
+def read_pyasdf(sfile: str, ccomp: str) -> Tuple[float, float, np.ndarray, np.ndarray]:
+    # useful parameters from each asdf file
+    with pyasdf.ASDFDataSet(sfile, mode="r") as ds:
+        alist = ds.auxiliary_data.list()
+        try:
+            dt = ds.auxiliary_data[alist[0]][ccomp].parameters["dt"]
+            dist = ds.auxiliary_data[alist[0]][ccomp].parameters["dist"]
+            print("working on %s (comp: %s) that is %5.2fkm apart. dt: %.3f " % (sfile, ccomp, dist, dt))
```

Comment:
Please use logging instead of printing. You can add this at the top:
```
import logging
logger = logging.getLogger(__name__)
```
```suggestion
            logger.info("working on %s (comp: %s) that is %5.2fkm apart. dt: %.3f " % (sfile, ccomp, dist, dt))
```